topo_processor/cog/create_cog.py

```diff
@@ -5,27 +5,27 @@
 from topo_processor.util.aws_credentials import Credentials, get_credentials
 
 
-def create_cog(input_path: str, output_path: str, compression_method: str) -> str:
+def create_cog(input_path: str, output_path: str, compression_method: str, overview_compression_method: str) -> str:
```

Comment:
Is there a reason to have an argument for this specific `gdal` parameter? i.e. are we anticipating something?

Reply:
IMO we generally should make this kind of change when it actually happens.

Reply:
Discussion with @dwsilk @paulfouquet - will hard code both `compression` and `overview_compression`.